client/pcr.go

```diff
@@ -98,6 +186,23 @@ func ReadAllPCRs(rw io.ReadWriter) ([]*pb.PCRs, error) {
 	return allPcrs, nil
 }
 
+// ReadAllPCRsDirect fetches all the PCR values from all implemented PCR banks.
+func ReadAllPCRsDirect(thetpm transport.TPM) ([]*pb.PCRs, error) {
```

Comment:
Same here

Reply:
Discussed offline, the naming convention will persist to maintain consistency.